test/appsignal/span_test.exs

```diff
@@ -180,16 +180,14 @@ defmodule AppsignalSpanTest do
   end
 
   describe ".set_name/2, with a span that doesn't have a reference" do
-    setup do
-      [return: Span.set_name(%Span{}, "test")]
-    end
+    setup :create_root_span
 
-    test "returns nil", %{return: return} do
-      assert return == nil
+    setup %{span: span} do
+      [span: span, return: Span.set_name(span, "test")]
     end
 
-    test "does not set the name through the Nif" do
-      assert Test.Nif.get(:set_span_name) == :error
+    test "returns the span", %{span: span, return: return} do
+      assert return == span
```

Comment:
I don't understand the changes to this test. Do we not want to still test that the name is not set through the Nif?

Does the added setup block do something I'm missing, compared to doing `assert span == Span.set_name(span, "test")`?